### PR TITLE
Adjust rollout scorer times

### PR DESCRIPTION
**Pull request by jonfortescue**
Closes dotnet/core-eng#13109.

When I wrote this I, out of cowardice, did configs twice and I always hated that so I finally fixed it while doing this.

Adjusted rollout scorer expected times based on averages of time to rollout.

### CODE REVIEW

File: src/RolloutScorer/RolloutScorer.Tests/RolloutScorerTests.cs [hunk 1]
```diff
@@ -39,10 +39,9 @@ public static IEnumerable<object[]> GetUriTestCases()
         public void RolloutScorerTests_SetUp()
         {
             _rolloutScorer = new RolloutScorer();
-            Config config = Utilities.ParseConfig();
-            _rolloutScorer.RolloutWeightConfig = config.RolloutWeightConfig;
-            _rolloutScorer.RepoConfig = config.RepoConfigs.First();
-            _rolloutScorer.AzdoConfig = config.AzdoInstanceConfigs.Find(a => a.Name == _rolloutScorer.RepoConfig.AzdoInstance);
+            _rolloutScorer.RolloutWeightConfig = StandardConfig.DefaultConfig.RolloutWeightConfig;
```
Comment (missymessa) [inline src/RolloutScorer/RolloutScorer.Tests/RolloutScorerTests.cs:42]: nit: initializations are cleaner looking this way: 

```
_rolloutScorer = new RolloutScorer 
{
    RolloutWeightConfig = StandardConfig.DefaultConfig.RolloutWeightConfig,
   ...
};
```

Comment (jonfortescue) [reply src/RolloutScorer/RolloutScorer.Tests/RolloutScorerTests.cs:42]: they really do...i love doing them that way, too........................................................................................................,,,,,,,,,,,,,,,,,,,,,,,,,,

**Comment by missymessa** [comment]
Jon and I chatted about other improvements, but we have an issue to refactor Rollout Scorer to improve DI/testability, so that will be covered then.

**Review by missymessa — APPROVED** [review]

**Comment by jonfortescue** [comment]
fyi @markwilkie

**Pull request merged**